src/detect-engine.c

```diff
@@ -1078,7 +1078,9 @@ void InspectionBufferInit(InspectionBuffer *buffer, uint32_t initial_size)
 void InspectionBufferSetupMulti(InspectionBuffer *buffer, const DetectEngineTransforms *transforms,
         const uint8_t *data, const uint32_t data_len)
 {
+#ifdef DEBUG_VALIDATION
```

Comment:
How is this related to HTTP byte ranges ?

Reply:
Its not, but i noticed it in my QA. Its in its own commit for that reason